### PR TITLE
fixed server register+login and create folder for db requests

### PR DESCRIPTION
**Pull request by didOstap**

### CODE REVIEW

File: server/src/db.requests/user.requests.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                throw new Error('Password did not update');
+            }
+
+            return updatedUser;
+        } catch (e) {
+            next(e);
+        }
+    };
+
+export const checkUserByProp = async (prop: string): Promise<IUserModel> => {
```
Comment (InnaKutsik) [inline server/src/db.requests/user.requests.ts]: checkUserByProp  is not the same as userExist  ?

File: server/src/models/token.model.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export interface ITokenModel extends Document {
 const TokenSchema: Schema = new Schema({
     user: {type: Schema.Types.ObjectId, ref: 'User', required: true},
     token: {type: String, required: true},
-    createdAt: {type: Date, default: Date.now, expires: 43200, required: true},
+    createdAt: {type: Date, default: Date.now, expires: 3600, required: true},
```
Comment (InnaKutsik) [inline server/src/models/token.model.ts]: expires move to config constant

File: server/src/controllers/confirm.user.controllers/confirm.token.controller.ts [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
-        await User.findOneAndUpdate(
-            { _id: token.user },
-            { isVerified: true },
-            { new: true });
+        const updatedUser = await verificateUser(token.user, next);
+        if (!updatedUser) {
+            throw new Error('User does not exist');
+        }
+
+        const removeToken = await Token.findByIdAndRemove(token.id);
```
Comment (InnaKutsik) [inline server/src/controllers/confirm.user.controllers/confirm.token.controller.ts]: move to server/src/db.requests/token.requests.ts

File: server/src/db.requests/user.requests.ts (hunk 1 above)
Comment (didOstap) [reply server/src/db.requests/user.requests.ts]: not the same
checkUserByProp has another usage in passport without "next"

Comment (InnaKutsik) [reply server/src/db.requests/user.requests.ts]: any way you need try/catch arrount await in order to catch error and handle it

**Pull request merged**